src/qa4sm_preprocessing/reading/imagebase.py

```diff
@@ -396,11 +396,10 @@ def repurpose(
         if (not (outpath / "grid.nc").exists()) or append:
             outpath.mkdir(exist_ok=True, parents=True)
             testimg = self._testimg()
-
             if (
                 drop_crs
                 and "crs" in self.varnames
-                and testimg.crs.dtype.type is np.string_
+                and testimg.crs.dtype.type is np.bytes_
```

Comment:
string_ not available, replaced with bytes_

Reply:
ok